src/Client.php

```diff
@@ -4,23 +4,32 @@
 
 use Ginger\ApiClient;
 use Ginger\Ginger;
+use GingerPluginSdk\Helpers\OrderBuilderTrait;
 use GingerPluginSdk\Properties\ClientOptions;
 
+require_once "./vendor/autoload.php";
+
 class Client
 {
-    private $api_client;
+    use OrderBuilderTrait;
+
+    protected ApiClient $api_client;
 
     public function __construct(ClientOptions $options)
     {
         require_once "./vendor/autoload.php";
```

Comment:
Auto loading should be arranged by the consumer of the library

Reply:
OK